src/client/src/containers/SelectBackendFramework/index.tsx

```diff
@@ -77,6 +77,7 @@ class SelectBackEndFramework extends React.Component<Props> {
       <div className={styles.container}>
         {options.length > 0 && (
           <SelectOption
+            isFrameworkSelection={true}
```

Comment:
> New prop `isFrameworkSelection` passed down to SelectableCards because this component is actually shared between all the pages, I only want the dependency info showing on the frameworks page

Reply:
You mean the component `SelectBackendFramworks` spans across multiple pages in the wizard?

Reply:
I think he means the selectoption component is used by all pages.

Reply:
Yeah, `SelectableCard` is shared among Pages and Framework selection, I needed the cards to somehow know it was framework selection

Reply:
I see, that makes sense. Thanks for explaining that :)